rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java

```diff
@@ -240,10 +240,12 @@ public AttachmentContent getAttachmentContent(String id) {
 
     public Resources<Resource<Attachment>> getResourcesFromList(Set<Attachment> attachmentList) {
         final List<Resource<Attachment>> attachmentResources = new ArrayList<>();
-        for (final Attachment attachment : attachmentList) {
-            final Attachment embeddedAttachment = restControllerHelper.convertToEmbeddedAttachment(attachment);
-            final Resource<Attachment> attachmentResource = new Resource<>(embeddedAttachment);
-            attachmentResources.add(attachmentResource);
+        if (attachmentList != null && !attachmentList.isEmpty()) {
```

Comment:
please  use `CommonUtils.isNotEmpty(attachmentList)`.

Reply:
Thanks @akapti for the review, i have used the CommonUtils.isNotEmpty(attachmentList).

Reply:
@ravi110336 : Kindly remove the null check, `CommonUtils.isNotEmpty` handles the null case.

Reply:
Thanks @akapti removed the extra null check.